hudi-spark/src/main/scala/org/apache/hudi/HoodieMergeOnReadRDD.scala

```diff
@@ -102,6 +104,44 @@ class HoodieMergeOnReadRDD(@transient sc: SparkContext,
     rows
   }
 
+  private def logFileIterator(split: HoodieMergeOnReadFileSplit,
+                             config: Configuration): Iterator[InternalRow] =
+  new Iterator[InternalRow] {
```

Comment:
this does not seem to be indented correctly?

Reply:
nice catch :)